sig-multicluster/annual-report-2022.md

```diff
@@ -4,66 +4,67 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-   -
-   -
-   -
+   - Significant progress on KEP-1645, the [MCS API](https://sigs.k8s.io/mcs-api) now with conformance tests
+   - Developed the About API for ClusterSets
 
 2. What initiatives are you working on that aren't being tracked in KEPs?
 
-   -
-   -
-   -
+   - SIG-MC is building a website to track and communicate multicluster related initiatives, patterns, and best practices. ([contribute](https://github.com/kubernetes-sigs/sig-multicluster-site))
+   - We are working with the Gateway API project to formalize interaction with multi-cluster services
+   - The SIG is exploring cross-cluster workload management - what might it take to support multicluster controllers? and the [work API](https://github.com/kubernetes-sigs/work-api)
 
 
 
 3. KEP work in 2022 (v1.24, v1.25, v1.26):
   - alpha:
     - [2149 - ClusterID for ClusterSet Identification](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/2149-clusterid) - v1.24
+  - progress to beta:
+      [1645 - MCS API](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api)
 
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
+   - There has been continued and repeated interest in multicluster controller development, but we need more people to come and share specific use cases, previous attempts at implementations, and/or concerns before we can hope to propose any generally useful solutions.
    -
    -
 
 2. What metrics/community health stats does your group care about and/or measure?
 
-   -
-   -
-   -
+   - We tend to focus most on meetup attendees and presenters, and contributors.
+     - In H2 2022 we averaged 15 attendees/week, now up to ~20 in 2023-03
+     - 2-3 presenters each week in 2023, up from ~1 in H2 2022
+   - We have regular discussion and repo contributions from at least 3 companies.
 
 3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
    to activities or programs that provide useful context or allow easy participation?
 
-   -
+   - NEEDS UPDATING
 
 4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
 
-   -
+   - NEEDS UPDATING
 
 5. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - Yes
```

Comment:
This doesn't include any work in K/K, but for repos owned by sig-mc, this dashboard shows # of orgs

https://k8s.devstats.cncf.io/d/8/company-statistics-by-repository-group?orgId=1&var-period=d7&var-metric=contributions&var-repogroup_name=SIG%20Multicluster&var-repo_name=kubernetes%2Fkubernetes&var-companies=All&viewPanel=1&from=1641016800000&to=1672466400000

It looks like there 5-6 active companies

Reply:
thanks, added the stat